### PR TITLE
#4 implement car prototype manager service

### PR DESCRIPTION
**Pull request by ZsoltToth**

### CODE REVIEW

File: cars/service/src/main/java/hu/uni/miskolc/iit/webdevelopment/examples/cars/dao/exceptions/PrototypeIDAlreadyExistsException.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package hu.uni.miskolc.iit.webdevelopment.examples.cars.dao.exceptions;
+
+/**
+ * Created by tothzs on 2017.11.24..
+ */
+public class PrototypeIDAlreadyExistsException extends Exception {
+}
```
Comment (judit0310) [inline cars/service/src/main/java/hu/uni/miskolc/iit/webdevelopment/examples/cars/dao/exceptions/PrototypeIDAlreadyExistsException.java]: Missing constructors

**Review by judit0310 — CHANGES_REQUESTED** [review]
Some issue point is not fulfilled.

File: cars/service/src/main/java/hu/uni/miskolc/iit/webdevelopment/examples/cars/dao/exceptions/PrototypeIDAlreadyExistsException.java (hunk 1 above)
Comment (ZsoltToth) [reply cars/service/src/main/java/hu/uni/miskolc/iit/webdevelopment/examples/cars/dao/exceptions/PrototypeIDAlreadyExistsException.java]: constructor has been added

**Review by ZsoltToth — COMMENTED** [review]
request is fulfilled

**Review by judit0310 — APPROVED** [review]

**Pull request merged**